### ISSUE TITLE
Claude A11y Test Bug

### ISSUE THREAD

**Issue by aanchalbhansali** [OneNoteDev/WebClipper#626]
Environment Details:
Application Name: OneNote Clipper-Web
OS: Windows 11 OS build 26058.1000
Edge Dev Version: Version 123.0.2400.1 (Official build) dev (64-bit)
AT's: Keyboard

Pre-Requisites:

Open the URL: [OneNote Web Clipper Installation](https://www.onenote.com/clipper)
Invoke the 'Get OneNote web Clipper for Microsoft Edge' button.
Edge Add-ons page will appear. Invoke the Get button.
OneNote Clipper will be added in edge extension section.
Expand the Extension button and turn on the OneNote Clipper Show in Toolbar button.
OneNote Clipper will appear beside of the search field.
Now invoke the OneNote clipper button.
Sign in with V-id.
Repro Steps:

Open the [Ankylosaurus - Wikipedia](https://en.wikipedia.org/wiki/Ankylosaurus)
Now invoke the OneNote clipper button.
OneNote Clipper dialog box will appear.
Tab to navigate to the Clip button and invoke it. And observe the issue.
Actual Result:
On invoking the Clip button, Keyboard focus is not visible on the Clip successful dialog box.
Note: Similar issue observe in the Region dialog box also.

Expected Result:
On invoking the Clip button, Keyboard focus should be visible on the Clip successful dialog box View in OneNote button.

User Impact:
Keyboard users might be confused, if there is no keyboard focus on the Successful dialog box.

MAS Reference:
[MAS 2.4.7 – Focus Visible](https://aka.ms/MAS2.4.7)

**Comment by aanchalbhansali** [comment]
@claude Can you create PR for this?
